styles/CTA.module.scss

```
@@ -0,0 +1,49 @@
+.link {
+    display: block;
+    background-color: var(--color--off-white);
+    padding: 20px;
+    max-width: $content--narrow;
+    margin: 0 auto $component-spacing--small;
+    transition: color $transition-quick, transform $transition-quick;
+
+    @include media-query(medium) {
+        margin: 0 auto $component-spacing--medium;
+    }
+
+    @include media-query(large) {
+        margin: 0 auto $component-spacing--large;
+    }
+
+    &:hover {
```

```suggestion
    &:hover,
    &:focus-visible {
```

Does the current styles setup do autoprefixing?